nomos-da/network/core/src/protocols/replication/mod.rs

```diff
@@ -171,6 +171,7 @@ mod test {
     }
 
     #[tokio::test]
+    #[ignore = "Create new fixtures"]
```

Comment:
@CHr15F0x , can you generate the new fixtures? Do we have a place which explains how to actually do this?